src/Classes/ServiceAPI/MyRadio_Podcast.php

```diff
@@ -792,14 +835,21 @@ public function convert()
     /**
      * Returns all Podcasts. Caches for 1h.
      *
+     * @param int $noResults The number of results to return per page. 0 for all podcasts.
+     * @param int $page The page required.
+     *
      * @return Array[MyRadio_Podcast]
      */
-    public static function getAllPodcasts()
+    public static function getAllPodcasts($noResults = 0, $page = 1)
```

Comment:
num_results instead? params tend to be snake_case, and "no" looks a bit like... "no"